scripts/vue_or_react.sh

```diff
@@ -4,9 +4,18 @@
 git fetch origin main
 react_count=$(git diff --name-only origin/main -- | grep "/clients/web/react/" | wc -l)
 vue_count=$(git diff --name-only origin/main -- | grep "/clients/web/vue3/" | wc -l)
+rn_count=$(git diff --name-only origin/main -- | grep "/clients/mobile/react-native/" | wc -l)
+config_file_path=""
 
 if [ $react_count -gt $vue_count ]; then
-  echo "cookiecutter/react_template.yaml"
+  config_file_path="cookiecutter/react_template.yaml"
 else
-  echo "cookiecutter/vue_template.yaml"
+  config_file_path="cookiecutter/vue_template.yaml"
 fi
+
+if [ "$rn_count" != 0 ]; then 
+  cp -r resources/ {{cookiecutter.project_slug}}/
+  sed -i.bak 's/include_mobile: "n"/include_mobile: "y"/' $config_file_path
+fi
+
+echo $config_file_path
```

Comment:
So you did improve this file then...
I don't love the use of sed here. I feel like it would be cleaner if this script just tells you what the focus of the PR is about and then with that info we pick which tempate to use from `cookiecutter/{script_result}_template.yaml`

Reply:
@oudeismetis  I tried to do the least to affect the rest of the builds I also did not want to remove building a client altogether because I do know there is a syntax bug when no client is selected (I believe one line loses too many spaces so the whole app crashes on building)